app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferencesImpl.kt

```diff
@@ -26,4 +27,13 @@ class MemoPreferencesImpl @Inject constructor(@ApplicationContext context: Conte
     override fun getPath(): String = sharedPreferences.getString(CURRENT_NOTES_PATH, "") ?: ""
 
     override fun getNotesStorage(): Path = Path(getPath())
+
+    override fun storeCrashReportEnabled(bool: Boolean) {
+        prefEditor.apply {
+            putBoolean(CRASH_REPORT_ENABLE, bool)
+            apply()
+        }
```

Comment:
Could this be simplified to one line of code as below?

```suggestion
prefEditor.putBoolean(CRASH_REPORT_ENABLE, bool).apply()
```